demos/demos_databases_apis/databricks_pyspark/graphistry-notebook-dashboard.ipynb

```diff
@@ -134,33 +114,53 @@
    },
    "outputs": [
     {
+     "output_type": "execute_result",
      "data": {
       "text/plain": [
-       "Out[12]: '0.28.5'"
+       "'0.34.3'"
       ]
      },
-     "metadata": {
-      "application/vnd.databricks.v1+output": {
-       "addedWidgets": {},
-       "arguments": {},
-       "data": "Out[12]: '0.28.5'",
-       "datasetInfos": [],
-       "metadata": {},
-       "removedWidgets": [],
-       "type": "ansi"
-      }
-     },
-     "output_type": "display_data"
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
     }
    ],
    "source": [
-    "import graphistry  # if not yet available, install and/or restart Python kernel using the above\n",
+    "import graphistry  # if not yet available, install pygraphistry and/or restart Python kernel using the cells above\n",
+    "graphistry.__version__"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "b5496fa5-525a-48c9-ad46-0ce17ebdc4f8",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [],
+   "source": [
     "\n",
-    "# To specify Graphistry account & server, use:\n",
-    "# graphistry.register(api=3, username='...', password='...', protocol='https', server='hub.graphistry.com')\n",
-    "# For more options, see https://github.com/graphistry/pygraphistry#configure\n",
+    "# Best practice:  use databricks secrets to store graphistry personal key (access token) \n",
     "\n",
-    "graphistry.__version__"
+    "# create personal key for your user at https://hub.graphistry.com/account/tokens\n",
+    "\n",
+    "graphistry.register(api=3, \n",
```

Comment:
do a markdown cell above explaining that we're using databrcik's native secrets manager here, and maybe a link on where to add their own secret